storage/src/cache/filecache/mod.rs

```diff
@@ -182,6 +183,14 @@ impl FileCacheEntry {
             .backend
             .get_reader(blob_info.blob_id())
             .map_err(|_e| eio!("failed to get blob reader"))?;
+        let rafs_blob_reader = if blob_info.rafs_blob_digest() != &[0u8; 32] {
```

Comment:
How about detecting the inline-bootstrap case by a flag from blob_info instead of checking digest is zero?

Reply:
Ok, how about adding a flag named `BLOB_FEATURE_INLINE`?